### DISCUSSION TITLE
Netdata without IPv6. Unbelievable

### DISCUSSION THREAD

**Discussion by hagodoy** [netdata/netdata#17075] [Q&A]
In the year 2024, IPv6 has not yet been implemented for app.netdata.cloud. I can't manage my servers that are IPv6 only. In 2020 I saw a post where someone complained about this. After 4 years nothing has been done and progressed. Is it that difficult? Unbelievable.

Not to mention the disaster that is GitHub without IPv6. Netdata should abandon GitHub for its project and move on to other solutions that adopt IPv6.

Any solution for my problem? Or will I have to look for other tools and abandon Netdata?

**Comment by ffrediani** [comment]
As I can see app.netdata.cloud is hosted in AWS which fully supports IPv6 so it might be that someone from the development just didn't look at that. What would take to have IPv6 enabled for the platform.
Is there anyone from the team who has actively looked at this ?

In 2024 there are no more reasons or barriers not to have and it becomes even more important when multiple hosting providers are starting to offer IPv6-only servers and VPS which need monitoring.

IPv6 is not cosmetic or some 'nice to have' thing anymore. It is the default Internet Protocol and as such should be available from day zero to any platform.

**Comment by hagodoy** [comment]
time=2024-02-27T16:20:53.682-03:00 comm=netdata source=daemon level=info tid=32032 thread=ACLK_MAIN msg="ACLK_OTP Got Password from Cloud"
time=2024-02-27T16:20:53.699-03:00 comm=netdata source=daemon level=error errno="101, Network is unreachable" tid=32032 thread=ACLK_MAIN msg="[mqtt_wss] E: Could not connect to remote endpoint \"mqtt.netdata.cloud\", port 443.\u000A"

fev 27 16:26:38 netdata[141582]: ACLK_OTP Got Password from Cloud
fev 27 16:26:38 netdata[141582]: [mqtt_wss] E: Could not connect to remote endpoint "mqtt.netdata.cloud", port 443.
fev 27 16:26:38 netdata[141582]: Connect failed
fev 27 16:26:40 netdata[141582]: HTTPS "GET" request to "app.netdata.cloud" finished with HTTP code: 200
fev 27 16:26:40 netdata[141582]: Getting Cloud /env successful
fev 27 16:26:41 netdata[141582]: HTTPS "GET" request to "api.netdata.cloud" finished with HTTP code: 429
fev 27 16:26:41 netdata[141582]: ACLK_OTP Challenge HTTP code not 200 OK (got 429)
fev 27 16:26:41 netdata[141582]: Error getting challenge
fev 27 16:26:41 netdata[141582]: Error passing Challenge/Response to get OTP

Server using IPv6 only
NAT64 was added and the connection did not work.
